code/game/gamemodes/gang/gang_datum.dm

```diff
@@ -134,7 +139,8 @@
 /datum/gang/proc/income()
 	if(!bosses.len)
 		return
-
+	if(fighting_style == "multiverse")
```

Comment:
IF YOU HAVE TO PUT STUFF LIKE THIS IN THE CODE, IT'S SNOWFLAKE CODE YOU'RE WRITING AND YOU NEED TO REFACTOR THE SYSTEMS TO BE GENERIC